src/types/index.ts

```diff
@@ -175,6 +176,7 @@ export interface IHiddenLayoutsExpressions {
 
 export interface IUiConfig {
   autoSave: boolean | null | undefined;
```

Comment:
Does any one know what this option is used for (autoSave). There isn't any documentation for it either.

Reply:
I'm pretty sure it's not in use. I searched around and never found any apps using it, but I've waited to get rid of the code in case I missed something.

Reply:
Now removed in #1227